tests/test_kernel_magics.py

```diff
@@ -53,19 +54,42 @@ def test_start_session():
     line = ""
     assert not magic.session_started
 
-    magic._do_not_call_start_session(line)
+    ret = magic._do_not_call_start_session(line)
```

Comment:
It's short for whatever is returned. Another very popular option is `ret_v`. Usually used in places where it's not really that meaningful, like a unit test.